orbs/shared/executors/testbed-machine.yml

```diff
@@ -1,7 +1,7 @@
 description: Standard executor for machine runtimes
 machine:
   image: ubuntu-2204:2022.10.2
-  docker_layer_caching: true
+  docker_layer_caching: false
```

Comment:
What are you doing to track the impact of this?

CircleCI says we use quite a bit of the layer cache in our services, which means this could cause substantial slowdown.

Reply:
Taking offline